lib/ProductOpener/Import.pm

```diff
@@ -54,8 +54,7 @@ It is also used in the C<scripts/import_csv_file.pl> script.
 
 package ProductOpener::Import;
 
-use utf8;
-use Modern::Perl '2017';
+use ProductOpener::PerlStandards
```

Comment:
The test is failing, probably because this line is missing a ; at the end

Reply:
Oops! I'll fix that